.gitmodules

```diff
@@ -51,7 +51,7 @@
 	url = https://github.com/nithinn/ncc.git
 [submodule "thirdparty/onednn_gpu"]
 	path = src/plugins/intel_gpu/thirdparty/onednn_gpu
-	url = https://github.com/oneapi-src/oneDNN.git
+	url = https://github.com/sungeunk/oneDNN.git
```

Comment:
I'm not OK to release from fork as we have open source types of distributions, where we have to update recipes for each OpenVINO release. Inside such distribution channels, owners will not allow such forks.

Reply:
Sorry for the confusion. This is not to be merged at this moment. We are trying to resolve the issue with OneDNN team. This is for CI checking and pre-validation purpose.